jmix-data/data/src/main/java/io/jmix/data/impl/JpqlQueryBuilder.java

```diff
@@ -265,8 +268,45 @@ protected void applyFiltering() {
                         .processParameters(resultParameters, queryParameters, actualized, entityName);
             }
 
+            Condition jpaOnlyCondition = removeNonJpaPropertyConditions(actualized);
```

Comment:
This is required when a filter for JPA entity contains conditions by non-JPA attributes. They should not be included in JPQL.